translations/ui-inventory/en.json

```diff
@@ -406,6 +406,7 @@
   "instanceHrid": "Instance HRID",
   "browseCallNumbers": "Browse call numbers",
   "browseSubjects": "Browse subjects",
+  "browseCallNumbers.missedMatch": "<strong>would be here</strong>",
```

Comment:
Best to move the `<strong>` tag to the mark-up, rather than have it in the translation string.